src/editor/asset_browser.cpp

```diff
@@ -302,7 +302,7 @@ bool AssetBrowser::resourceInput(const char* label, const char* str_id, char* bu
 		m_wanted_resource = buf;
 	}
 	ImGui::SameLine();
-	ImGui::Text(label);
+	ImGui::Text("%s", label);
```

Comment:
gcc can check string format strings and arguments and gives warning (which becomes error with -Werror)

```
../../../src/editor/asset_browser.cpp: In member function ‘bool AssetBrowser::resourceInput(const char*, const char*, char*, int, Lumix::uint32)’:
../../../src/editor/asset_browser.cpp:305:19: error: format not a string literal and no format arguments [-Werror=format-security]
  ImGui::Text(label);
                   ^
cc1plus: all warnings being treated as errors
```
If you pass printf-like function not a string literal but some received string, this function can misinterpret it (suppose this string has `%s` in it) and read arbitrary memory.